projects/laji/src/app/+taxonomy/taxon/taxon-tree/tree/tree.component.html

```diff
@@ -3,7 +3,7 @@
     <ng-container *ngFor="let node of tree.parentNodeList; let last = last">
       <div *ngIf="!node.state.isSkipped || node.value.id === activeId">
         <laji-spinner *ngIf="last && node.state.loadingCount > 0"></laji-spinner>
-        <ng-container *ngTemplateOutlet="labelTpl; context: {'value': node.value}"></ng-container>
+        <ng-content /><ng-container *ngTemplateOutlet="labelTpl; context: {'value': node.value}"></ng-container>
```

Comment:
This is unnecessarily hacky. We weren't able to reproduce the issue, but it appears that this is what is happening:
- `taxon-tree.component` passes the label template as content projection
- `tree.component.ts`: `@ContentChild` gets a reference to that template, but this behavior appears to be buggy (maybe it gets the template only after the tree has been rendered so the tree doesn't have access to it on the first render? idk)
- adding `<ng-content/>` actually does the content projection. Perhaps this ensures that the template is indeed available? Note that even if this works we are getting the reference to the template twice, and it's really hard to tell if everything is working correctly since that would require some deeper knowledge of angular internal magic

Better solution: Don't use content projection on `ng-template`s. Instead pass a reference to the template as a `@Input`. I created a new branch for this (avoid cluttering commit history, and the package.json change had issues too).  https://github.com/luomus/laji/pull/471